.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+# https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        # Test with the latest LTS.
+        node: ['14']
+    runs-on: ubuntu-latest
```

Comment:
I'm leaning things about travis right up to the end -- `lts/*` is "the latest node LTS version": https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ 👍